packages/prosess-vedtak/src/components/VedtakForm.jsx

```diff
@@ -264,7 +264,11 @@ const transformValues = values =>
   values.aksjonspunktKoder.map(apCode => ({
     kode: apCode,
     begrunnelse: values.begrunnelse,
-    fritekstbrev: { brødtekst: values.brødtekst, overskrift: values.overskrift },
+    fritekstbrev: {
+      overstyrtMottaker: safeJSONParse(values.overstyrtMottaker),
```

Comment:
overstyrtMottaker kan gjelde for både autogenererte vedtaksbrev og fritekstvedtaksbrev. Den må sendes alene til dokumentdata med koden `dokumentdatatype.OVERSTYRT_MOTTAKER`. Det gir derfor kanskje ikke mening å ha den i fritekstbrev-objektet?

Reply:
Ah, tolket det som at valg av mottaker bare skulle vises når man overstyrte med fritekst.

Reply:
Stemmer at det kun er fritekst på unntaksløypen - sånn sett riktig tolket. 
Funksjonaliteten for overstyring av mottaker vil også benyttes på hovedløypen til omsorgspenger, og der er det både automatisk og fritekst.